sources/packages/backend/workflow/test/2024-2025/parttime-assessment/appeals/parttime-assessment-student-additional-transportation.e2e-spec.ts

```diff
@@ -15,18 +15,23 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-student-addition
       // Arrange
       const assessmentConsolidatedData =
         createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
-      assessmentConsolidatedData.studentDataEligibleForAnAdditionalTransportationAllowance =
+      assessmentConsolidatedData.studentDataAdditionalTransportRequested =
         YesNoOptions.Yes;
+      assessmentConsolidatedData.studentDataAdditionalTransportListedDriver =
+        YesNoOptions.Yes;
+      assessmentConsolidatedData.studentDataAdditionalTransportOwner =
+        YesNoOptions.No;
       assessmentConsolidatedData.studentDataAdditionalTransportKm = 30;
       assessmentConsolidatedData.offeringWeeks = 20;
       assessmentConsolidatedData.studentDataAdditionalTransportWeeks = 15;
-      assessmentConsolidatedData.studentDataAdditionalTransportCost = 30;
       assessmentConsolidatedData.studentDataAdditionalTransportPlacement =
         YesNoOptions.No;
 
       assessmentConsolidatedData.appealsStudentAdditionalTransportationAppealData =
         {
-          eligibleForAnAdditionalTransportationAllowance: YesNoOptions.Yes,
+          additionalTransportRequested: YesNoOptions.Yes,
+          additionalTransportListedDriver: YesNoOptions.Yes,
+          additionalTransportOwner: YesNoOptions.Yes,
           additionalTransportKm: 12,
           additionalTransportCost: 111,
```

Comment:
Is the `additionalTransportCost` still part of the appeals or should it be removed?